docs/source/privacy/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Data Collection"
+sidebar_title: "Data Collection"
+description: "Information about data collected by Rover"
+---
+
+By default, Rover collects some anonymous usage data to help better understand the needs of our users. 
+
+## Collected Data
+
+Rover reports a few different pieces of information each time you run a command:
+
+- The command that was run _(excluding any identifiable arguments such as file-system paths or profile names)_.
+- The version of `rover` that was executed.
+- A unique, anonymized "machine identifier," which is the same for every command run on the same machine.
```

Comment:
Bold **machine identifier** and **session identifier** instead of putting in quotes